### PR TITLE
unstructured grid gen

### PR DESCRIPTION
**Pull request by lnw**
a short script to generate unstructured grids for visualisations. it has a help menu :)

### CODE REVIEW

File: tools/unstructured_grid_gen.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+#!@PYTHON_EXECUTABLE@
+#
```
Comment (bast) [inline tools/unstructured_grid_gen.py:2]: are these top three lines needed?

File: tools/unstructured_grid_gen.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+sys.path.append('@PYTHON_INSTDIR@')
+from QCTools import QCTools
+from argparse import ArgumentParser
+
+def parseCommandline():
+    parser = ArgumentParser(prog="unstructured_grid_gen.py", description="first create a body centred cubic lattice, then prune using atomic coordinates", add_help=True)
+    parser.add_argument("--coord-file",   help="",      nargs=1, dest="coord_file", action="store", type=str, required=False, default=['coord'])
+    parser.add_argument("--cutoff",       help="[a_0]", nargs=1, dest="cutoff",   action="store", type=float,  required=False, default=5.0)
+    # parser.add_argument("--cutoff-perpB", help="[a_0]", nargs=1, dest="cutoff_pB",  action="store", type=float,  required=False, default=8.0 )
+    # parser.add_argument("--B",            help="",      nargs=3, dest="B",          action="store", type=float,  required=False, default=[0, 0, 1] )
```
Comment (bast) [inline tools/unstructured_grid_gen.py]: please remove out-commented code

**Comment by bast** [comment]
Thanks!

**Comment by lnw** [comment]
Thanks! That was wonderfully fast.

**Pull request merged**